src/webgpu/api/validation/capability_checks/limits/maxStorageTexturesPerShaderStage.spec.ts

```diff
@@ -130,6 +135,13 @@ g.test('createPipeline,at_over')
       limitTest,
```

Comment:
Same comment as above with these tests w.r.t using a param for async and not async. But feel free to do it in another change if it gets too messy in this one or if there are other similar cases in other limit tests.

Reply:
it was pretty messy but it was a good to cleanup